src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,131 @@
 package juja.sqlcmd;
 
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
 public class Application {
-    public static void main(String[] args) {
+
+    private static final String POSTGRESQL_DRIVER = "org.postgresql.Driver";
+    private static final String DB_CONNECTION_PATH = "jdbc:postgresql://127.0.0.1:5432/sqlcmd";
+    private static final String DB_USER = "sqlcmd";
+    private static final String DB_PASSWORD = "sqlcmd";
+    private static final String TABLE_NAME = "user";
+    public static final String SHOW_EXISTING_TABLES_NAME = "SELECT * FROM pg_catalog.pg_tables WHERE schemaname != 'pg_catalog' AND schemaname != 'information_schema';";
```

Comment:
You can implement it without inversions as "SELECT table_name FROM information_schema.tables WHERE table_schema='public' AND table_type='BASE TABLE'" .
And why is it constant?

Reply:
It is constant because we will never change the order or schema name for this functionality.